sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBSuite.scala

```diff
@@ -588,28 +822,6 @@ class RocksDBSuite extends SparkFunSuite {
         verifyMetrics(putCount = 2, getCount = 3, metrics = db.metrics)
       }
     }
-
-    // force compaction and check the compaction metrics
```

Comment:
Can we retain this part of test and be selectively executed for incremental checkpoint? Please try not to remove the test where the functionality is not removed.

Reply:
(The config is even applied to the snapshot of changelog checkpointing, although we do not do that in every microbatch hence not easy to test that. OK to skip test for changelog checkpointing.)

Reply:
Added back and test with changelog checkpointing disabled.